src/construction.jl

```diff
@@ -102,7 +109,7 @@ function create_nodes(node, state, profile, static)::Dictionary{Int, ODESystem}
     emptysys = ODESystem(Equation[], t, [], []; name = :empty)
     sysdict = Dictionary{Int, ODESystem}(node.id, fill(emptysys, nrow(node)))
     # create all node systems
-    for node in eachrow(node)
+    for node in Tables.rows(node)
```

Comment:
Does this also return a DataFrameRow, but with the advantage that it also supports other tables?

Reply:
Yes x2 👍🏻